crates/re_query/src/query.rs

```diff
@@ -119,7 +122,7 @@ pub fn query_archetype<A: Archetype>(
     store: &DataStore,
     query: &LatestAtQuery,
     ent_path: &EntityPath,
-) -> crate::Result<ArchetypeView<A>> {
+) -> crate::Result<(Option<TimeInt>, ArchetypeView<A>)> {
```

Comment:
Can we put the time in the `ArchetypeView` instead, to reduce the amount of tuples everywhere? It would also allow us to document what the time means (most recent component, when is it `None`, etc)